components/atoms/Tile/Tile.scss

```
@@ -0,0 +1,28 @@
+.uic--tile {
+  background-color: $white;
+
+  &--dark {
+    background-color: $midnight;
+
```

```suggestion
    background-color: $midnight;
    color: $white
```
I think we should make the text white by default when dark

---

I'm ok with that